configure.ac

```diff
@@ -4728,34 +4731,34 @@ then
         ENABLED_CERTEXT="yes"
         AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_CERT_EXT"
     fi
-    
+
     # Requires CRL
     if test "x$ENABLED_CRL" = "xno"
     then
         ENABLED_CRL="yes"
         AM_CFLAGS="$AM_CFLAGS -DHAVE_CRL"
     fi
-    
-	if test "x$ENABLED_SRP" = "xno"
-	then
+
+    if test "x$ENABLED_SRP" = "xno"
+    then
         ENABLED_SRP="yes"
-	    AM_CFLAGS="$AM_CFLAGS -DWOLFCRYPT_HAVE_SRP"
-	fi
-	
+            AM_CFLAGS="$AM_CFLAGS -DWOLFCRYPT_HAVE_SRP"
+        fi
```

Comment:
Move this line and above back an indent.

Reply:
:heavy_check_mark: